sonarts-core/tests/rules/noNestedTemplateLiteralsRule/noNestedTemplateLiteralsRule.lint.ts

```diff
@@ -0,0 +1,51 @@
+function fooNestedTemplateLiterals() {
```

Comment:
@andrea-guarino-sonarsource you can use "nok" instead of "foo" prefix for noncompliant and "ok" instead of "bar" for compliant